roles/ansible/tower/manage-inventories/tasks/process-inventory.yml

```diff
@@ -90,4 +96,5 @@
   set_fact:
     org_id: ''
     inv_id: ''
+    inventory_output_create: ''
```

Comment:
it's good to reset this one (although unnecessary with how the latest functionality of register works in Ansible) , but if we do for the `create` output, please also do for `update`